example-ha-ip-pool-global-ip-example-multi-metro.tfvars

```diff
@@ -0,0 +1,8 @@
+metal_auth_token = "redacted"
```

Comment:
The examples should each get their own directory in the `example` directory (e.g., this one could be in a directory named `example/ha-op-pool-global-ip-multi-metro/`); the example should include a `main.tf` that consumes the module. The directory structure for examples is demonstrated here: https://github.com/equinix-labs/terraform-equinix-template/tree/main/examples/simple

Reply:
Added to the docs better